Keyboard/Views/KeyView.swift

```diff
@@ -349,8 +412,8 @@ final class KeyView: UIView {
     }
 
     private func setupFullStop(_ traits: UITraitCollection, _ page: KeyboardPage) {
-        if UIDevice.current.dc.isIpad && traits.userInterfaceIdiom == .pad {
-            if (UIDevice.current.dc.screenSize.sizeInches ?? 0) < 12.0 {
+        if Device.current.isPad && traits.userInterfaceIdiom == .pad {
+            if Device.current.diagonal < 12.0 {
```

Comment:
We probably want these kinds of magic numbers to be constants for iOS 28 when they invent a rhombus sized iPad